### PR TITLE
recipe-samples: add a packagegroup and an image for LKFT

### PR DESCRIPTION
**Pull request by fboudra**
The LKFT image is close to the LAVA image with 2 main differences:
* ptest isn't enabled in the image features
* contains additional packages provided by the LKFT package group
* include packagegroup-core-tools-debug

Signed-off-by: Fathi Boudra <fathi.boudra@linaro.org>

### CODE REVIEW

File: recipes-samples/packagegroups/packagegroup-rpb-tests.bb [hunk 1]
```diff
@@ -12,6 +12,7 @@ PACKAGES = "\
 # contains basic dependencies, that don't need graphics/display
 RDEPENDS_packagegroup-rpb-tests = "\
     packagegroup-core-buildessential \
+    packagegroup-core-tools-debug \
```
Comment (ndechesne) [inline recipes-samples/packagegroups/packagegroup-rpb-tests.bb]: tools-debug is 'expected' to be an IMAGE_FEATURE. the fact that tools-debug maps to packagegroup-core-tools-debug is an 'implementation' detail, we should problably not rely on it.

FEATURE_PACKAGES_tools-debug = "packagegroup-core-tools-debug"

**Comment by ndechesne** [comment]
with the exception of the minor remark i made, I am fine with this change. let's try to keep the delta as small as possible (and we can close #124 then)

**Comment by koenkooi** [comment]
I would *strongly* recommend to forbid using IMAGE_FEATURES, it breaks the expectation that the image recipe describes the image contents. And worse, the meaning of the features can change on the whim of OE-core maintainers, so you only know after the fact what's in the image.

**Comment by fboudra** [comment]
Abandoned. Comments fixed in https://github.com/96boards/meta-rpb/pull/152

**Pull request closed without merging**